packages/@aws-cdk/aws-sagemaker-alpha/lib/model.ts

```diff
@@ -206,6 +206,17 @@ export interface ModelProps {
    * @default true
    */
   readonly allowAllOutbound?: boolean;
+
+  /**
+   * Whether to isolate to enable network isolation for the model container.
```

Comment:
I messed up, sorry: `Whether to enable network isolation for the model container.`